app/src/main/java/com/nervousfish/nervousfish/activities/VisualVerificationActivity.java

```diff
@@ -28,9 +28,11 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger("VisualVerificationActivity");
     private static final int SECURITY_CODE_LENGTH = 5;
+    private static final int NUM_BUTTONS = 12;
```

Comment:
Everybody knows how many tiles there are on the screen so they can figure out that the picture's animals and objects aren't randomly placed and can more easily seatup for a brute force attack.